rfcs/rdm-0077-vocabulary-harvesting.md

```diff
@@ -246,6 +250,14 @@ Multi-step operation:
 
 #### Updating of relations
 
+#### Deprecation Behavior 
+
+When a property is marked as deprecated, it will be excluded from search results and cannot be selected for new records or modifications of existing records. The `deprecated_by` field is optional and can be used to indicate one or more values that supersede the deprecated property. Note that populating the `superseded_by` field does not affect existing records; those records will remain unchanged.
```

Comment:
There are some use cases where we might want to be able to select to deprecated values, for this, we allow to add deprecate values through the API for instance or and admin shuold be able to do it.

- [ ] Add a new tag "non-searchable" (or find a better name) that will prevent the values from appearing in the searches.
- [ ] Permission based, to allow admins to curate the records, only admins shuold see the deprecated values.
- [ ] just add the tag for the moment. no extra system fields.